ctest.h

```diff
@@ -16,6 +16,10 @@
 #ifndef CTEST_H
 #define CTEST_H
 
+#if !(defined (_GNU_SOURCE) || defined (_BSD_SOURCE) || defined(_POSIX_C_SOURCE))
+#undef CTEST_SEGFAULT
```

Comment:
That could be a good idea. From the documentation, kill() should also be present all almost all modern systems. Chris, did you run into a problem with this or is your proposal to make it more correct?